scripts/install_third_party.py

```diff
@@ -25,6 +25,7 @@
 import sys
 import tarfile
 import zipfile
+import urllib.request as urlrequest
```

Comment:
Do not use `as` just import `urllib` and use that.

Reply:
Actually, I tried that but it showed the error ` urllib has no attribute 'request'`.

Reply:
@DubeySandeep Can you PTAL!